goalie/go_mesh_seq.py

```diff
@@ -187,7 +187,9 @@ def indicate_errors(
             # Loop over each strongly coupled field
             for f in self.fields:
                 # Loop over each timestep
-                for j in range(len(self.solutions[f]["forward"][i])):
+                solutions = self.solutions.extract(layout="field")
+                indicators = self.indicators.extract(layout="field")
```

Comment:
More of a question for my education: why not just do `self.solutions.extract()`?

Reply:
A couple of reasons:
* So that the source code explains much of what it is doing without further comments.
* To avoid an issue if the default layout gets changed later on.

Reply:
Thanks! :)